docs/api/kernelspec.rst

```diff
@@ -30,6 +30,10 @@ kernelspec - discovering kernels
       Additional kernel-specific metadata; clients can use this as needed,
       for instance to aid in kernel selection and filtering.
 
+      As with all other metadata fields, jupyter clients should use
```

Comment:
Clients as in any _jupyter protocol_ frontend, not necessarily a jupyter _branded_ client.